FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -5,6 +5,34 @@
 import Foundation
 
 public final class RemoteFeedLoader: FeedLoader {
+	
+	private struct Root: Codable {
+		let items: [ImageItem]
+		var feed: [FeedImage] {
+			return items.map { $0.feedImage }
+		}
+	}
+
+	private struct ImageItem: Codable {
```

Comment:
Please see my comment about `Codable` above.

```suggestion
	private struct ImageItem: Decodable {
```